addons/website/views/snippets.xml

```diff
@@ -14,7 +14,7 @@
     </section>
```

Comment:
The arrows on the block_slider.png image are misplaced... the left one is against the snippet, the right one is across.